src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonIgnoreCondition.cs

```diff
@@ -5,21 +5,22 @@
 namespace System.Text.Json.Serialization
 {
     /// <summary>
-    /// Controls how the <see cref="JsonIgnoreAttribute"/> ignores properties on serialization and deserialization.
+    /// When specified on <see cref="JsonSerializerOptions.DefaultIgnoreCondition"/>, controls whether properties with default values are ignored during serialization.
```

Comment:
As written, "controls whether" is required because it's referring to a set of options; "specifies that" would work with a specific option. Could rephrase to something like `When specified on <see cref="JsonSerializerOptions.DefaultIgnoreCondition"/>, WhenWritingDefault specifies that properties with default values are ignored during serialization.`